src/commands/appdistribution-distribute.ts

```diff
@@ -98,11 +98,11 @@ module.exports = new Command("appdistribution:distribute <distribution-file>")
       );
     }
 
-    const releaseHash = await distribution.releaseHash();
+    let binaryName = await distribution.binaryName(app);
```

Comment:
Why is this `binaryName` re-assigned?  Should it be two variables?

Reply:
Good question. It should be the same value, so I think using the same variable (name) makes sense here.

Some background:
The `binaryName` is based on the project number, app ID, and content (sha-256 hash) of the file being uploaded. Here it's calculated on the client side as an optimization/shortcut to check whether the binary has been uploaded successfully already. If it hasn't been uploaded successfully previously, then we upload it. The server then returns the `binaryName` determined on the server-side (line 114). We use that (which should be identical) to check the upload status.

Reply:
If it's not identical would that be indicative of some problem we'd want to tell the user about or at least log?

Reply:
I don't think we want to tell the user about it, because it's not actionable. Logging would be an option, but the logs are on the user's machine and so we'd not really learn about any problems.

Ideally we'd remote log (maybe using Google Analytics) that there's a mismatch which could alert us. But I have difficulties imagining scenarios where this really matters. In the worst case the shortcut simply doesn't work.